README.md

```diff
@@ -1 +1,131 @@
-# Hello-Microverse
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="images/coder.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is created by HTML and CSS to be showed a green colored Hello Microverse Title.
+
+
+## 🛠 Built With <a name="built-with"></a>
+Html and CSS
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[CSS file is externally linked to HTML]**
+- **[h1 tag is colored to green]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Code editor(VS), Web browser and Git.
+
+
+### Setup
+
+Clone this repository to your desired folder: https://github.com/kakalygaday/Hello-Microverse
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command: 
+Open the file with your default brower or use live server extension
+
+Example command:
+
+```sh
+  rails server
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdy**
+
+- GitHub: [@githubhandle](https://github.com/kakalygaday)
+
+
+- [ ] **[Change the color of h1 tag]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project put a star and follow me for more.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank for these extra busy days.
```

Comment:
- [ ] You have done really great job here, however, as requested by the previous reviewer, Please mention anyone or a group of people that inspired you or guided you in implementing the project instead of saying: `I would like to thank for these extra busy days.`  Acknowledgment expresses gratitude towards individuals or organizations who have contributed to the project. It shows appreciation for their time, effort, and expertise, fostering a positive and collaborative community around the software. 🙂